server/controllers/utils.ts

```diff
@@ -378,33 +378,49 @@ createOneTeam = (teamSize: number, numRounds: number, people: any[]) => {
   return rounds;
 };
 
-export const createDynamicTeams = (teamSize: number, numRounds: number) => {
+export const createDynamicTeams = (teamSize: number, numRounds: number, dynamicOptions: boolean) => {
   /*
   * Returns array of {roundPairs, roundGen}
   * e.g.:
   * {roundPairs: [[1,6], [2,3], [4,5]], roundGen: {<some roundGen with 50% - n, 50% - 1 teamSize structure>}}
   * first round in pair is always with 1 user in a team*/
   const availableNumbers = Array.from(Array(numRounds).keys());
-  const roundPairs = randomPairs(availableNumbers);
+  const roundPairs = consecutivePairs(availableNumbers);
+  console.log("HERE ARE MY ROUNDPAIRS: " + roundPairs)
```

Comment:
Cute but perhaps something that gives more context to future users? :-) Feel free to adjust to something. you prefer, but this could work: 
```suggestion
  console.log("Round pairs: " + roundPairs)
```